Core/Repositories/IPostRepository.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Core.Entities;
+
+namespace Core.Repositories
+{
+    public interface IPostRepository : IRepository<Post>
+    {
+        Task CreateComment(Comment comment);
+        Task<List<Post>> GetPostsForBlog();
+        Task<List<Post>> GetPostsForNotes(Guid userId);
```

Comment:
Weird interface to have in the repository, do you think it is possible to move it up into a "service" layer?